megatron/training/training.py

```diff
@@ -1150,7 +1150,7 @@ def track_e2e_metrics():
                 'validation_iterations_time_msecs_avg': validation_iterations_time_msecs_avg
             })
 
-    while iteration < args.train_iters:
+    while iteration < args.train_iters and iteration < args.force_stop_iter:
```

Comment:
```suggestion
    while iteration < args.train_iters and (args.force_stop_iter is None or iteration < args.force_stop_iter):
```
現状だとforce_stop_iterがNoneだとエラーを吐くと思うので修正しました。